Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx

```diff
@@ -20,14 +20,18 @@ import {
   DetailsRow,
 } from 'office-ui-fabric-react/lib/DetailsList';
 import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
-import { BotTemplate } from '@bfc/shared';
+import { BotTemplate, creationFeatureFlagReadMe } from '@bfc/shared';
```

Comment:
Since this URL is temporary, lets just put it inline in here instead of shared.  fewer places to clean up!